features/ouders.feature

```diff
@@ -272,7 +272,7 @@ Rule: Wanneer de geslachtsnaam van de ouder onbekend is, wordt de ouder geleverd
     | burgerservicenummer | 999993008 |
     En de persoon heeft een 'ouder' met de volgende gegevens
     | naam                                            | waarde   |
-    | ouderAanduiding.code                            | ouder1   |
+    | ouderAanduiding.code                            | 1        |
```

Comment:
ouderAanduiding is geen waardetabel meer, maar een string. Dus niet "ouderAanduiding.code", maar "ouderAanduiding"